graphql/database/missions/updateMissionNotification.js

```diff
@@ -0,0 +1,23 @@
+import UserMissionModel from './UserMissionModel'
+
+/**
+ * Records a user's closing the "Mission Completed" notification.
+ *
+ * @param {object} userContext - The user authorizer object.
+ * @param {string} userId - The user's ID.
+ * @param {string} missionId - The mission's ID.
+ */
+const updateMissionNotification = async (userContext, userId, missionId) => {
+  // Get the mission model and modify it.
+  await UserMissionModel.update(userContext, {
```

Comment:
instead of using UserContext here, you'll want to use
```
import {
  getPermissionsOverride,
  MISSIONS_OVERRIDE,
} from '../../utils/permissions-overrides'

const override = getPermissionsOverride(MISSIONS_OVERRIDE)
```
and pass in the override as the first argument to UserMissionModel.update

Reply:
Ohh